Dockerfile

```diff
@@ -18,7 +18,7 @@
 ######################################################################
 # Node stage to deal with static asset construction
 ######################################################################
-ARG PY_VER=3.9-slim-bookworm
+ARG PY_VER=3.10.13-slim
```

Comment:
Should we pin to the micro version here or just point to the minor like we did before? There are pros/cons here, but we may miss out on security updates, and we don't have a process like dependabot to monitor and bump this. I'd say we stick to the previous strategy to point to the minor release image.

Reply:
To clarify, I'd vote to point to `3.10-slim-bookworm` here. It looks available here https://hub.docker.com/_/python